src/Filter/Setting/CustomSql.php

```diff
@@ -429,21 +425,21 @@ private function convertParameterAggregate($var, $arguments)
 
         $this->addParameters($var);
 
-        return rtrim(str_repeat('?,', count($var)), ',');
+        return \rtrim(\str_repeat('?,', \count($var)), ',');
     }
 
     /**
      * Convert a parameter in the query string.
      *
-     * @param array $arrMatch The match from the preg_replace_all call in parseRequestVars().
+     * @param string $strMatch The match from the preg_replace_all call in parseRequestVars().
      *
      * @return string
      *
      * @internal Only to be used via parseRequestVars().
      */
-    public function convertParameter($arrMatch)
+    public function convertParameter($strMatch)
```

Comment:
Can be made private now and the `@internal` can be removed. This method is now being called from `$this->checkTag()`.